### PR TITLE
MS dynamics destination updates

### PR DESCRIPTION
**Pull request by mohan-h**

### CODE REVIEW

File: help/destinations/catalog/crm/microsoft-dynamics-365.md [hunk 1]
```diff
@@ -80,13 +84,19 @@ The [Requests limits and allocations](https://docs.microsoft.com/en-us/power-pla
 
 {style="table-layout:auto"}
 
+## Supported audiences {#supported-audiences}
```
Comment (vfalon) [inline help/destinations/catalog/crm/microsoft-dynamics-365.md:87]: Please check with engineering whether this destination supports external audiences or not.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md [hunk 2]
```diff
@@ -38,7 +38,11 @@ Go to the [!DNL Dynamics 365] [trial](https://dynamics.microsoft.com/en-us/dynam
...
 
 Create the custom field of type `Simple` with field data type as `Single Line of Text` which Experience Platform will use to update the audience status within [!DNL Dynamics 365].
-Refer to the [!DNL Dynamics 365] documentation to [create a field (attribute)](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) if you need additional guidance.
+
+Refer to the [!DNL Dynamics 365] [Create or edit a field (attribute)](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) documentation if you need additional guidance.
+
+The part of the name of any custom field you create in [!DNL Dynamics 365] is the **[!UICONTROL Customization prefix]**. Note down this value as it is required during the [Fill in destination details](#destination-details) step. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details.
```
Comment (vfalon) [inline help/destinations/catalog/crm/microsoft-dynamics-365.md]: "The part of the name of any custom field you create"

I don't really understand what this means. Can you please rephrase this to be clearer?

File: help/destinations/catalog/crm/microsoft-dynamics-365.md [hunk 3]
```diff
@@ -122,6 +132,7 @@ To configure details for the destination, fill in the required and optional fiel
 
 * **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 * **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+* **[!UICONTROL Customization Prefix]**: The part of the name of any custom field you create in [!DNL Dynamics 365] is the customization prefix. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details.
```
Comment (vfalon) [inline help/destinations/catalog/crm/microsoft-dynamics-365.md]: "The part of the name of any custom field you create"

I don't really understand what this means. Can you please rephrase this to be clearer?

File: help/destinations/catalog/crm/microsoft-dynamics-365.md [hunk 4]
```diff
@@ -133,7 +144,7 @@ When you are finished providing details for your destination connection, select
 
 >[!IMPORTANT]
 >
->To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Audiences]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
```
Comment (vfalon) [inline help/destinations/catalog/crm/microsoft-dynamics-365.md]: Please revert this change. The permission names have not been changed.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md [hunk 5]
```diff
@@ -199,10 +210,10 @@ To validate that you have correctly set up the destination, follow the steps bel
 1. Switch to the **[!DNL Activation data]** tab, then select an audience name.
 ![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/crm/microsoft-dynamics-365/destinations-activation-data.png)
 
-1. Monitor the audience summary and ensure that the count of profiles corresponds to the count created within the segment.
-![Platform UI screenshot example showing Segment.](../../assets/catalog/crm/microsoft-dynamics-365/segment.png)
+1. Monitor the audience summary and ensure that the count of profiles corresponds to the count created within the audience.
+![Platform UI screenshot example showing Audience.](../../assets/catalog/crm/microsoft-dynamics-365/segment.png)
```
Comment (vfalon) [inline help/destinations/catalog/crm/microsoft-dynamics-365.md]: No need for capital A in Audience.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md [hunk 6]
```diff
@@ -224,3 +235,18 @@ To fix this error, verify that the **[!UICONTROL Mapping ID]** you provided in [
...
+
+### Changelog
+
+This section captures the functionality and significant documentation updates made to this destination connector.
+
++++ View changelog
+
+|Release month|Update type|Description|
+|---|---|---|
+|August 2023|Functionality update| We fixed an issue where the user is unable to leverage a [!DNL Dynamics 365] custom field if it is not created within the default solution. An input field, **[!UICONTROL Customization Prefix]** has been introduced in the [Fill in destination details](#destination-details) step where the user would input the required prefix. (PLATIR-31602) |
```
Comment (vfalon) [inline help/destinations/catalog/crm/microsoft-dynamics-365.md]: This is both a functionality AND a documentation update, so please update that section too.

Also, let's rephrase and split this update to be easier to understand:

- Fixed an issue where users were unable to use a [!DNL Dynamics 365] custom field if it was not created within the default solution.
- A new input field, **[!UICONTROL Customization Prefix]**, has been added in the [Fill in destination details](#destination-details) step, where users must input the required prefix.

Also, I don't really understand if these two updates are related to each other.

- What does "default solution" mean?
- What does the prefix do? 
- Is there a link between the prefix and the default solution or custom field? 

Please review this section and add some clarity around what this means for the customers.

**Review by vfalon — CHANGES_REQUESTED** [review]
Thanks for the update, @mohan-h . There are some things to clarify here, still. Please see my comments.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 2 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: It's the same phrasing as given by MSD so as to not create confusion. Have done a minor rewording. Refer to the associated image shown immediately in the next line (45).

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 3 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: Its the same phrasing as given by MSD so as to not create confusion. Have done a minor rewording.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 6 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: Updated the changelog.  The queries you raise are clarified by going through _Create field within [!DNL Dynamics 365] {#prerequisites-custom-field}. Terminology is specific to MSD. To elaborate again would be replicating their writeup.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 1 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md:87]: It doesn't support external audiences as there is a dependency on MSD's contactID.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 2 above)
Comment (vfalon) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: The explanation is too convoluted and I doubt a new reader understand what that is.
Let's rephrase this section as such:

"Write down the [!UICONTROL Customization prefix] of the custom field that you create in [!DNL Dynamics 365]. You will need this prefix during the [Fill in destination details](#destination-details) step. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details."

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 3 above)
Comment (vfalon) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: Rephrase this as such:

"The [!UICONTROL Customization prefix] of the custom field that you created in [!DNL Dynamics 365]. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details."

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 6 above)
Comment (vfalon) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: Let's rephrase this to be even clearer:

- Added support for [!DNL Dynamics 365] custom field prefixes. A new input field, **[!UICONTROL Customization Prefix]**, has been added in the [Fill in destination details](#destination-details) step.
- Fixed an issue where users were unable to use a custom field if it was not created within the default solution in [!DNL Dynamics 365].

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 3 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: Also changed the [!UICONTROL Customization prefix] to `Customization prefix` for the description.

File: help/destinations/catalog/crm/microsoft-dynamics-365.md (hunk 6 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/microsoft-dynamics-365.md]: Both bullets are the same have changed to 

Added support for [!DNL Dynamics 365] custom field prefixes for custom fields which were not created within the default solution in [!DNL Dynamics 365]. A new input field, **[!UICONTROL Customization Prefix]**, has been added in the [Fill in destination details](#destination-details) step. (PLATIR-31602).
